### PR TITLE
Allowing any key (or a specified set of keys) to trigger keyboard navigation mode, allowing to specify a set of dissmiss keyboard navigation mode keys.

### PR DESCRIPTION
**Pull request by mshoho**

### CODE REVIEW

File: src/Keyborg.ts [hunk 1]
```diff
@@ -88,12 +93,27 @@ class KeyborgCore implements Disposable {
...
+  private _dismissKeys?: Set<number>;
 
-  constructor(win: WindowWithKeyborg) {
+  constructor(win: WindowWithKeyborg, props?: KeyborgProps) {
     this.id = "c" + ++_lastId;
     this._win = win;
     const doc = win.document;
 
+    if (props) {
+      const triggerKeys = props.triggerKeys;
```
Comment (ling1726) [inline src/Keyborg.ts:105]:
```suggestion
      const triggerKeys = props.triggerKeys ?? [];
```

That way you can get rid of the `if` clauses

File: src/Keyborg.ts [hunk 2]
```diff
@@ -232,8 +258,8 @@ export class Keyborg {
   private _core?: KeyborgCore;
   private _cb: KeyborgCallback[] = [];
 
-  static create(win: WindowWithKeyborg): Keyborg {
-    return new Keyborg(win);
+  static create(win: WindowWithKeyborg, props?: KeyborgProps): Keyborg {
```
Comment (ling1726) [inline src/Keyborg.ts:261]:
```suggestion
  static create(win: WindowWithKeyborg, props: KeyborgProps = {}): Keyborg {
```
Default options can always just be empty

File: src/Keyborg.ts [hunk 3]
```diff
@@ -247,7 +273,7 @@ export class Keyborg {
     instance._cb.forEach((callback) => callback(isNavigatingWithKeyboard));
   }
 
-  private constructor(win: WindowWithKeyborg) {
+  private constructor(win: WindowWithKeyborg, props?: KeyborgProps) {
```
Comment (ling1726) [inline src/Keyborg.ts:276]:
```suggestion
  private constructor(win: WindowWithKeyborg, props: KeyborgProps = {}) {
```

File: src/Keyborg.ts [hunk 4]
```diff
@@ -320,8 +346,8 @@ export class Keyborg {
   }
 }
 
-export function createKeyborg(win: Window): Keyborg {
-  return Keyborg.create(win);
+export function createKeyborg(win: Window, props?: KeyborgProps): Keyborg {
```
Comment (ling1726) [inline src/Keyborg.ts:349]:
```suggestion
export function createKeyborg(win: Window, props: KeyborgProps = {}): Keyborg {
```

File: src/Keyborg.ts (hunk 2 above)
Comment (mshoho) [reply src/Keyborg.ts:261]: Plus it renders to a few more bytes of JS to create that object (so, it won't be cheaper than `if (props)`).

**Review by ling1726 — APPROVED** [review]

**Pull request merged**